test/plugin/base_test.rb

```diff
@@ -2292,9 +2420,22 @@ def verify_subfields_when_not_hash(payload_key)
       d.run
     end
     verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
-      field = get_fields(entry['jsonPayload'])[payload_key]
-      assert_equal 'a_string', get_string(field), entry
-      assert_false entry.key?(destination_key), entry
+      if payload_key == DEFAULT_LABELS_KEY
+        # The malformed field has been removed from the payload.
+        assert_true get_fields(entry['jsonPayload']).empty?, entry
+        # No additional labels.
+        assert_equal COMPUTE_PARAMS[:labels].size,
+                     entry[destination_key].size, entry
+      else
+        # Leave the malformed field as it is.
+        # TODO(qingling128) On the next major after 0.7.4, make the rest of
```

Comment:
Would it be better to have this TODO in the implementation, rather than in the test?

Reply:
Done

Reply:
I meant in the actual implementation, in `out_google_cloud.rb`, probably in `set_log_entry_fields`.

Reply:
Sync'ed offline. Added TODO in both places.